admin_guide/scheduling/scheduler.adoc

```diff
@@ -21,38 +21,208 @@ particular node.
+            "name": "MaxGCEPDVolumeCount"
+        },
+        {
+            "name": "MaxAzureDiskVolumeCount"
+        },
+        {
+            "name": "MatchInterPodAffinity"
+        },
+        {
+            "name": "NoDiskConflict"
+        },
+        {
+            "name": "GeneralPredicates"
+        },
+        {
+            "name": "PodToleratesNodeTaints"
+        },
+        {
+            "name": "CheckNodeMemoryPressure"
+        },
+        {
+            "name": "CheckNodeDiskPressure"
+        },
+        {
+            "name": "NoVolumeNodeConflict"
+        },
+        {
+            "argument": {
+                "serviceAffinity": {
+                    "labels": [
+                        "region"
+                    ]
+                }
+            },
+            "name": "Region"
+
+         }
+    ],
+    "priorities": [
+        {
+            "name": "SelectorSpreadPriority",
+            "weight": 1
+        },
+        {
+            "name": "InterPodAffinityPriority",
+            "weight": 1
+        },
+        {
+            "name": "LeastRequestedPriority",
+            "weight": 1
+        },
+        {
+            "name": "BalancedResourceAllocation",
+            "weight": 1
+        },
+        {
+            "name": "NodePreferAvoidPodsPriority",
+            "weight": 10000
+        },
+        {
+            "name": "NodeAffinityPriority",
+            "weight": 1
+        },
+        {
+            "name": "TaintTolerationPriority",
+            "weight": 1
+        },
+        {
+            "argument": {
+                "serviceAntiAffinity": {
+                    "label": "zone"
+                }
+            },
+            "name": "Zone",
+            "weight": 2
+        }
+    ]
+}
+----
+
+[[modifying-scheduler-policy]]
+=== Modifying Scheduler Policy
+
+The scheduler policy is defined in a file on the master,
+named *_/etc/origin/master/scheduler.json_* by default,
+unless overridden by the `kubernetesMasterConfig.schedulerConfigFile`
+field in the
+xref:../../install_config/master_node_configuration.adoc#master-configuration-files[master configuration file].
+
+.Sample modified scheduler configuration file
+[source,json]
+----
+kind: "Policy"
+version: "v1"
+predicates:
+  - name: "PodFitsResources"
+  - name: "NoDiskConflict"
+  - name: "MatchNodeSelector"
+  - name: "HostName"
+priorities:
+  - name: "LeastRequestedPriority"
+    weight: 1
+  - name: "BalancedResourceAllocation"
+    weight: 1
+  - name: "ServiceSpreadingPriority"
+    weight: 1
+----
+
+
+To modify the scheduler policy:
+
+. Edit the scheduler configuration file to configure the desired
+xref:default-scheduler-policy[default predicates and priorities].
+You can create a custom configuration, or use and modify one of the
+xref:scheduler-sample-policies[sample policy configurations].
+
+. Add any xref:configurable-predicates[configurable predicates] and xref:configurable-priorities[configurable-priority-functions]
```

Comment:
for preview version, priority-function have wrong link, so please give a check here